service/src/test/java/bio/terra/landingzone/stairway/flight/create/CreateLandingZoneResourcesFlightIntegrationTest.java

```diff
@@ -0,0 +1,142 @@
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.retry.annotation.EnableRetry;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+@Tag("integration")
+@ActiveProfiles("test")
+@PropertySource("classpath:integration_azure_env.properties")
+@ExtendWith(SpringExtension.class)
+@SpringBootTest()
+@SpringBootApplication(
+    scanBasePackages = {
+      "bio.terra.common.logging",
+      "bio.terra.common.migrate",
+      "bio.terra.common.kubernetes",
+      "bio.terra.common.stairway",
+      "bio.terra.landingzone"
+    },
+    exclude = {DataSourceAutoConfiguration.class})
+@EnableRetry
+@EnableTransactionManagement
+public class CreateLandingZoneResourcesFlightIntegrationTest extends LandingZoneTestFixture {
+
+  @Mock private BearerToken bearerToken;
+
+  @Autowired LandingZoneService landingZoneService;
+
+  @Autowired
+  @Qualifier("landingZoneStairwayComponent")
+  StairwayComponent stairwayComponent;
+
+  UUID jobId;
+  UUID landingZoneId;
+  ProfileModel profile;
+  LandingZoneRequest request;
+  LandingZoneManager landingZoneManager;
+
+  @BeforeAll
+  static void init() {
+    Awaitility.setDefaultPollInterval(Duration.ofSeconds(5));
+  }
+
+  @BeforeEach
+  void setup() {
+
+    jobId = UUID.randomUUID();
+    landingZoneId = UUID.randomUUID();
+
+    profile =
+        new ProfileModel()
+            .managedResourceGroupId(resourceGroup.name())
+            .subscriptionId(UUID.fromString(azureProfile.getSubscriptionId()))
+            .tenantId(UUID.fromString(azureProfile.getTenantId()))
+            .cloudPlatform(CloudPlatform.AZURE)
+            .description("dummyProfile")
+            .id(UUID.randomUUID());
+    request =
+        new LandingZoneRequest(
+            StepsDefinitionFactoryType.CROMWELL_BASE_DEFINITION_STEPS_PROVIDER_TYPE.getValue(),
+            "v1",
+            Map.of(),
+            profile.getId(),
+            Optional.of(landingZoneId),
+            true);
+    landingZoneManager =
+        LandingZoneManager.createLandingZoneManager(
+            tokenCredential, azureProfile, resourceGroup.name());
+  }
+
+  @Test
+  void createResourcesFlightDeploysCromwellResources() throws Exception {
+    String resultPath = "";
+    landingZoneService.startLandingZoneResourceCreationJob(
+        jobId.toString(), request, profile, landingZoneId, bearerToken, resultPath);
+
+    await()
+        .atMost(Duration.ofMinutes(20))
+        .untilAsserted(
+            () -> {
+              var flightState = retrieveFlightState(jobId.toString());
+              assertThat(flightState.getFlightStatus(), not(FlightStatus.RUNNING));
+            });
+    var flightState = retrieveFlightState(jobId.toString());
+    assertThat(flightState.getFlightStatus(), oneOf(FlightStatus.READY, FlightStatus.SUCCESS));
```

Comment:
I'd think that `SUCCESS` would be the only acceptable flight status here, `READY` appears to mean the flight hasn't run yet